docs/guides/fetch-data/read-profile-data.md

```diff
@@ -359,158 +341,124 @@ fetchReceiverData().then((receiverAddress) => console.log(receiverAddress));
 
 Below is the complete code snippet of this guide, with all the steps compiled together.
```

Comment:
I suggest to not do this but to put this in a GitHub repo instead.

Because here it is annoying to maintain.